main/eval/src/mill/eval/EvaluatorImpl.scala

```diff
@@ -14,6 +14,7 @@ import scala.reflect.ClassTag
  */
 private[mill] case class EvaluatorImpl(
     home: os.Path,
+    projectRoot: os.Path,
```

Comment:
```suggestion
    workspaceRoot: os.Path,
```

We should try to use the same term to establish some domain language.

Reply:
Then I would just call it `workspace`. What do you think?

Reply:
Sound good.